CTFd/challenges.py

```diff
@@ -36,7 +36,7 @@ def chals():
             if view_after_ctf():
                 pass
             else:
-                return redirect('/')
+                return redirect(url_for('/'))
```

Comment:
You actually don't want to specify the root in ``config.py`` - you want that to be handled by uwsgi.  If you specify ``APPLICATION_ROOT`` it causes issues with the session cookie